server/routes/index.js

```diff
@@ -1,8 +1,10 @@
+/* eslint-disable import/no-unresolved */
```

Comment:
Please remove this and edit the eslint config to implement this

Reply:
It has been implemented